.github/workflows/publish-sdk.yml

```diff
@@ -29,6 +29,17 @@ jobs:
           node-version: 18.12.1
       - run: npm ci
       - run: npm run build
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=short_ref::${GITHUB_REF#refs/*/}
+
+      - name: Check output
+        run: echo ${{ steps.vars.outputs.short_ref }}
+      - run: |
+          if [[ ${{ steps.vars.outputs.short_ref }} == "feature/232-sdk-deployment" ]]; then
```

Comment:
`if [[ ${GITHUB_REF#refs/*/} }} == "feature/232-sdk-deployment" ]]; then`
This would be much simpler than creating and moving a variable round